src/main/java/Ingredient.java

```diff
@@ -0,0 +1,4 @@
+public class Ingredient {
+    String ingredientName;
+    int ingredientAmount;
```

Comment:
Nie powtarzaj w nazwie pól nazwy klasy: Jesteśmy w klasie Ingredient, i wszystkie pola do niej należą. Do zmiany nazwy możesz użyć skrótu Shift+F6, wtedy zmienią się wszystkie miejsca gdzie to było używany.